coin_flip/live.html

```diff
@@ -0,0 +1,77 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- A real extension would not be able to link to external CDNs; we just do it here for simplicity -->
+    <link rel= "stylesheet" type = "text/css" href = "./Coinflipstyle.css">
+    <!--
+      Twitch only allows loading assets from certain white-listed domains. Above we reference a font
+      from Google"s font CDN. Here we load the latest version of Muxy"s JavaScript SDK. You can
+      reference specific versions by replacing `latest` with a version number (e.g. `2.0.0`), or keep
+      this as-is to automatically load the most recent version.
+    -->
+    <script src="https://ext-cdn.muxy.io/medkit/latest/medkit.umd.js"></script>
+
+    <!--
+      Here is the entire code block needed to power the panel. It uses Muxy"s state loading to get
+      the last sent message, and easy event binding to get notified when a new message is sent.
+    -->
+    <script type="text/javascript">
+
+      /*
+      * Here we are giving the SDK a hint that this will run on the broadcaster"s dashboard and
+      * have the same rights a broadcaster does. This is only needed and used when developing an
+      * extension. Once it is deployed to production, the backend will use the correct rights from
+      * the extension system. You can leave this line in and it will not adversely affect the
+      * final extension.
+      */
+      Muxy.testJWTRole = "broadcaster";
+
+      Muxy.setup({ clientID: "replaceme" });
+      const sdk = new Muxy.SDK();
+
+      
+      var x = 0;
+
+      function startGame() {
+        sdk.send("start");
+          x = 0;
+          document.querySelector("#status").innerText = "Game started";
+          var mytimer = setInterval(getData,1000);
+          //Here we receive the message "start" sent through Muxy's SDK.
+
+          function getData() {
+            document.querySelector("#results").style.opacity = 1;
+            if (x == 1){
+              clearInterval(mytimer);
+            }
+            //This is where and how we stop the setInterval.
+
+            sdk.getRankingData("WinLose").then(function(arrayx) { //arrayx is just a variable name. 
+              console.log(arrayx) // The Muxy SDK is returning the viewer data and printing it out inside the console.
+              var winners = arrayx.find(function(win) { return win.key === "win" });
+              var losers = arrayx.find(function(lose) { return lose.key === "lose" });
+              document.querySelector("#results").innerText = "Winners: " + winners.score + "\n Losers: " + losers.score;
+              //Displaying the number of winner and losers.
+            })
+          } 
+      } 
+
+      function endGame() {
+        x = x + 1;
+        document.querySelector("#status").innerText = "Game ended";
+        sdk.send("end");
+        // Here is the function that will run when the "End game" is clicked.
```

Comment:
Comment should really go above the function and explain its purpose. Also it would be clearer to store the `mytimer` variable outside of the start and end functions and clear it here, rather than using the intermediary `x`.